fgspectra/frequency.py

```diff
@@ -100,7 +71,7 @@ def __call__(self, nu=None, beta=None, nu_0=None):
         """
         beta = np.array(beta)[..., np.newaxis]
         nu_0 = np.array(nu_0)[..., np.newaxis]
-        return (nu / nu_0)**beta
+        return (nu / nu_0)**beta * _rj2cmb(nu)
```

Comment:
Since we are suppoting only one unit, for time being we can hardcode it in the `eval`s
(there is a bug, though, here: we also need `/_rj2cmb(nu_0)`